### PR TITLE
Republish world location news when a feature changes

### PR DESCRIPTION
**Pull request by brucebolt**
When a featured document changes, we need to republish the `WorldLocationNews` content to make this change live on the website.

As these pages were previously rendered by Whitehall, it did not matter that this didn't happen.  But now we render these pages in Collections using the content item, this update is essential.

[Trello card](https://trello.com/c/9FzBvL38/282-bug-featuring-a-document-on-world-location-news-doesnt-update-on-the-live-site)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/controllers/admin/features_controller.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def create
     end
 
     if @feature.save
-      @feature.republish_organisation_to_publishing_api
+      @feature.republish_featurable_to_publishing_api
```
Comment (JonathanHallam) [inline app/controllers/admin/features_controller.rb:16]: So when this happens are we republishing the featured thing or the world location news page that they're featured on? I'd hope there'd be no need for the former but 'featurable' is an ambiguous name...

Comment (brucebolt) [reply app/controllers/admin/features_controller.rb:16]: The `featurable` is the page that the item is featured on (i.e. the World Location News or Organisation page).  I chose this name as it matches the database field.

Comment (JonathanHallam) [reply app/controllers/admin/features_controller.rb:16]: oh yeah, I've seen it before, not a complaint about your use of it, more its lack of description

**Review by JonathanHallam — APPROVED** [review]
👍

**Pull request merged**